### PR TITLE
Revise and parallelize 'evaluate' CLI and re-run to include in DVC

### PR DESCRIPTION
**Pull request by mdekstrand**
This revises the `evaluate` command-line options to take one NAME option, and use it to generate all 3 filenames, and adds the ability to measure recommendations in parallel.

Re-runs the evaluation to include the latest results in the DVC-generated files.

### CODE REVIEW

File: src/poprox_recommender/evaluation/evaluate.py [hunk 1]
```diff
@@ -27,14 +28,21 @@
...
-from poprox_concepts.api.recommendations import RecommendationRequest
 from poprox_concepts.domain import Article, ArticleSet
+from poprox_recommender.config import available_cpu_parallelism
 from poprox_recommender.data.mind import MindData
 from poprox_recommender.evaluation.metrics import rank_biased_overlap
 from poprox_recommender.logging_config import setup_logging
 from poprox_recommender.paths import project_root
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("poprox_recommender.evaluation.evaluate")
```
Comment (mdekstrand) [inline src/poprox_recommender/evaluation/evaluate.py:38]: executable scripts need this, because __name__ will be __main__ when they are run.

**Review by sophiasun0515 — APPROVED** [review]
Looks good to me! I also pulled the changes and run locally with name input and verified metrics result and running time is good.

**Comment by mdekstrand** [comment]
Great, I'll merge once the dvc run outputs are committed.

**Comment by mdekstrand** [comment]
definitely need more changes here (see slack)

**Pull request merged**